modules/apps/commerce/commerce-order-content-web/test/js/tests_utilities/utils.spec.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import userEvent from '@testing-library/user-event';
+
+import '@testing-library/jest-dom/extend-expect';
+import {fireEvent} from '@testing-library/react';
+import {act} from 'react-dom/test-utils';
+
+export async function setFieldValue(field, value, checkValue = true) {
+	if (field === null) {
+		return;
+	}
+
+	if (field instanceof HTMLSelectElement) {
+		userEvent.selectOptions(field, String(value));
+	}
+	else {
+		if (String(value).length) {
+			await userEvent.type(field, String(value));
+			field.value = String(value);
+		}
+		else {
+			field.value = '';
+		}
+	}
+	act(() => {
```

Comment:
Just a reminder that `act` is deprecated under testing-library, but since React 18, it's importable directly from `'react'` for tests. This non-blocking as we're still running tests under react 16 nonetheless.